### PR TITLE
fix bug from cleanup

### PR DESCRIPTION
**Pull request by adizaimi**
Fixing bug from a previous cleanup in https://github.com/bloomberg/comdb2/pull/853

### CODE REVIEW

File: bdb/file.c [hunk 1]
```diff
@@ -7753,9 +7753,11 @@ static int bdb_process_unused_files(bdb_state_type *bdb_state, tran_type *tran,
                 rc = bdb_get_file_version_data(bdb_state, tran, i, &version_num,
                                                bdberr);
             }
-            if (rc == 0 && version_num == file_version) {
-                found_in_llmeta = 1;
-                break;
+            if (rc == 0) {
```
Comment (riverszhang89) [inline bdb/file.c]: Why?

Comment (adizaimi) [reply bdb/file.c]: Putting back to how old code was...nothing fancy here, just correcting my mistake. Thanks.

Comment (riverszhang89) [reply bdb/file.c]: Thanks!

**Comment by ldennis** [comment]
This looks good to me. However, (probably) due to some issues introduced in #884, it is not passing tests yet.

**Comment by adizaimi** [comment]
master branch passes. This branch not rebased to the latest master, also passes the failing tests above.
Adding checkin from #884 seems to start causing the abort to be called (rep_record.c:2968: transaction failed at xxx).

**Pull request merged**